packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -13,8 +13,12 @@ export function renderInputComponent(props) {
 	const { caret, key, debounceMinLength, debounceTimeout, icon, inputRef, ...rest } = props;
 
 	const hasIcon = icon || caret;
+	const typeaheadContainerIconClasses = classNames(
+		hasIcon && theme['typeahead-input-icon'],
+		caret && theme['typeahead-input-caret'],
+	);
```

Comment:
please do not remove `tc-typeahead-typeahead-input-icon` classname.

Reply:
I will restore the default class name